Hub & Spoke concepts updated, removed unsiupported statements (#62309)
MicrosoftDocs/azure-docs: articles/azure-vmware/concepts-avs-hub-and-spoke-integration.md

-   **Azure VMware Solution private cloud:** Azure VMware Solution SDDC formed by one or more vSphere clusters, each one with a maximum of 16 nodes.

-   **ExpressRoute gateway:** Enables the communication between Azure VMware Solution private cloud, shared services on Hub virtual network, and workloads running on Spoke virtual networks.

-   **ExpressRoute Global Reach:** Enables the connectivity between on-premises and Azure VMware Solution private cloud.

  > [!NOTE]

>   > **S2S VPN considerations:** For Azure VMware Solution production deployments, Azure S2S VPN is not supported due to network requirements for HCX. However, it can be used for a PoC deployment.

Reviewer: ```suggestion
  > **S2S VPN considerations:** For Azure VMware Solution production deployments, Azure S2S VPN is not supported due to network requirements for VMware HCX. However, it can be used for a PoC deployment.
```

    -   **PaaS Spoke:** A PaaS Spoke hosts Azure PaaS services using private addressing thanks to [Private Endpoint](../private-link/private-endpoint-overview.md) and [Private Link](../private-link/private-link-overview.md).

-   **Azure Firewall:** Acts as the central piece to segment traffic between the Spokes and Azure VMware Solution.

-   **Application Gateway:** Exposes and protects web apps that run either on Azure IaaS/PaaS or Azure VMware Solution virtual machines (VMs). It integrates with other services like API Management.

## Network and security considerations

ExpressRoute connections enable traffic to flow between on-premises, Azure VMware Solution, and the Azure network fabric. Azure VMware Solution uses [ExpressRoute Global Reach](../expressroute/expressroute-global-reach.md) to implement this connectivity.

> On-premises connectivity must use ExpressRoute Global Reach as well in order to be able to communicate the on-premises vSphere environment with AVS since an ExpressRoute Gateway does not provide transitive routing between its connected circuits. 

Reviewer: ```suggestion
On-premises connectivity must use ExpressRoute Global Reach to communicate with the on-premises vSphere environment with Azure VMware Solution since an ExpressRoute gateway does not provide transitive routing between its connected circuits. 
```

> [!IMPORTANT]
> A route with address prefix 0.0.0.0/0 on the **GatewaySubnet** setting is not supported.

Set routes for specific networks on the corresponding route table. For example, routes to reach Azure VMware Solution management and workloads IP prefixes from the spoke workloads and vice versa.

:::image type="content" source="media/hub-spoke/specify-gateway-subnet-for-route-table.png" alt-text="Set routes for specific networks on the corresponding route table":::

A second level of traffic segmentation using the network security groups within the Spokes and the Hub to create a more granular traffic policy.

> [!NOTE]

> > **Traffic from on-premises to AVS:** Trafficc between on-premises workloads, either vSphere based ones or other, will be enabled thanks to Global Reach, but that traffic will not go through Azure Firewall on the hub. Therefore customer must implement traffic segmentation mechanisms either on-premises or on AVS.

Reviewer: ```suggestion
> **Traffic from on-premises to Azure VMware Solution:** Traffic between on-premises workloads, either vSphere based ones or other, will be enabled thanks to Global Reach, but that traffic will not go through Azure Firewall on the hub. Therefore the customer must implement traffic segmentation mechanisms either on-premises or on Azure VMware Solution.
```
